packages/framework/presence/api-report/presence.alpha.api.md

```diff
@@ -37,10 +37,12 @@ export interface IPresence {
 
 // @alpha @sealed
 export interface ISessionClient<SpecificSessionClientId extends ClientSessionId = ClientSessionId> {
-    connectionId(): ClientConnectionId;
-    getStatus(): SessionClientStatus;
+    getConnectionId(): ClientConnectionId;
+    getConnectionStatus(): SessionClientStatus;
     // (undocumented)
     readonly sessionId: SpecificSessionClientId;
+    setConnectionId(connectionId: ClientConnectionId): void;
+    setConnectionStatus(connected: boolean): void;
```

Comment:
Add a changeset. Any public API change should be in the release notes, even for experimental packages or alpha APIs.

Reply:
The changeset should cover the whole API update since 2.4 since I don't think there was a changeset update for earlier changes.